homeassistant/components/cast/media_player.py

```diff
@@ -593,6 +593,15 @@ async def async_play_media(self, media_type, media_id, **kwargs):
                 return
 
         # Default to play with the default media receiver
+        if media_type == "application/vnd.apple.mpegurl":
```

Comment:
This is to mimic the `camera.play_stream` service. But still can't get it to work.